drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -842,7 +870,8 @@ function DragAndDropBlock(runtime, element, configuration) {
 
             initDraggable();
             initDroppable();
-
+            pageLoaded = true
```

Comment:
Is there some other way to do this which doesn't require setting a (sort of) global variable?

e.g. by checking for the presence of a loaded page element?